src/tensors/gpu/cudnn_wrappers.cu

```diff
@@ -170,17 +170,24 @@ void ConvolutionWrapper::setConvDescriptor(int hPad,
                                            int wStride) {
   CUDNN_CALL(cudnnCreateConvolutionDescriptor(&convDesc_));
 
+#if CUDNN_MAJOR > 5
```

Comment:
If this only differs in one parameter, consider doing the #if only for that section, as it will be easier for the reader to see where it differs. Seems that's what the old version was?